### PR TITLE
Add in Tutorial on Replicating and Browsing

### PR DESCRIPTION
**Pull request by ebarry**
hi folks, i did another pass to get closer.

### CODE REVIEW

**Comment by dcwalk** [comment]
This looks great! I'm just going to add in the Sidebar/summary stuff, hopefully this afternoon we can get some content in and rebuild

**Comment by b5** [comment]
@ebarry is working on scaffolding for the first & second lessons, both visible in the branch. 

@b5 suggested using a "happy path" version that serves fewer edge cases, including reference to the "deeper" distributed web primer that includes more details. 

@ebarry needs help with a second lesson on getting the hash of data. one of us should connect to make sure all concepts are clear.

File: add-dataset/README.md [hunk 1]
```diff
@@ -62,11 +56,13 @@ Adding, harvesting, and storing are all verbs used in Data Together to describe
 
 Data Together helps groups steward data they care about by [...]
 
+* Public Record means [...]
+* Data Together Nodes means [...]
+* Distributed Data Stewardship means [...]
```
Comment (b5) [inline add-dataset/README.md]: The definition of these may come up in conversation at the 4S event, but generally we should put time into defining these!

**Comment by b5** [comment]
Quick potential definitions for tutorial:

**The Public Record**: a shared record of the information that has been published by the plethora of entities on the web
**Data Together Nodes means**: Let's leave this definition off for now, as Data Together nodes don't yet exist
**Distributed Data Stewardship** is using distributed technologies to secure & provide access to data that you care about.

File: browse-datasets/README.md [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
 ****
 
-### Step 1.
+### Step 1. Add metadata
 
-* [...]
+* Return to your web browser, and signup/signin to http://archivers.co/
 
-### Step 2.
+* Navigate to the content page for `CAFO_Density.zip`: http://archivers.co/content/12208b35b5c819ef6a8bb5df4657acbed1fbf084f8efe3174b3896db0058d91e82e5
```
Comment (b5) [inline browse-datasets/README.md]: Can we use this url instead? http://archivers.co/content/QmQU7RoBd8FWAkbePif7Xw5cjoL1Wb6wxCefdTuoA7XGa7

Comment (mhucka) [reply browse-datasets/README.md]: OK, done in commit 4b0e9eb

**Review by dcwalk — APPROVED** [review]
Great! Whoever merges will need to rebuild locally and push to gh-pages with the bash script!

**Review by mhucka — APPROVED** [review]
This is great work by @ebarry and really moved the ball forward on this tutorial.

**Comment by mhucka** [comment]
@ebarry I'm not deleting branch liz-test-branch in case you want to keep it around, but if you don't, can you go ahead and delete it?

**Pull request merged**